app/views/components/item-card/subitems.js

```diff
@@ -32,10 +35,13 @@ var SubItem = React.createClass({
     ev.preventDefault();
     let node = this.refs.addItemInput.getDOMNode();
     let title = node.value;
-    ItemActions.addItem(this.props.productId, {
+    let productId = this.getParams().id;
+    let storyId = this.getParams().number;
```

Comment:
It was something that I had seen elsewhere so I was using it to be consistent. Wasnt aware they were being deprecated.

Switched to use props